spaceship/lib/spaceship/connect_api/models/device.rb

```diff
@@ -106,9 +108,9 @@ def self.devices_for_platform(platform: nil, include_mac_in_profiles: false, cli
         end
 
         filter = {
-          status: Spaceship::ConnectAPI::Device::Status::ENABLED,
-          platform: device_platforms.uniq.join(',')
+          status: Spaceship::ConnectAPI::Device::Status::ENABLED
         }
+        filter[:platform] = device_platforms.join(',') unless device_platforms.empty?
```

Comment:
we don't need the `uniq` anymore?

Reply:
Forgot about this. Will fix in a couple of minutes